src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
+
+In learning settings hackathons might be held as part of course curriculum or as an extracurricular activity for interested students. Academic institutions might seek to partner with community groups for hackathons to encourage students and faculty to form connections outside the institution.
+
+### Hacking
+
+We use the term "hacking" expansively here to refer to ideas of in-depth learning by doing, do-it-yourself and <a href="https://en.wikipedia.org/wiki/Hacker_culture" target="_blank" class="link-external">"playful cleverness"</a> that we can also find present in the term "making". We feel (though others might disagree) that the concepts have enough overlap to discuss them together.
+
+We also consider the hacking approach to be present in one-to-one customizations, adaptations and transformations of technology that allow participation in activities or the use of technology by more diverse groups, including people with disabilities. This approach is found in groups like <a href="http://www.tetrasociety.org/" target="_blank" class="link-external">The Tetra Society of North America</a>, who create customized assistive devices for individuals.
+
+In learning settings, hacking might include classroom projects involving prototyping a product or system, or customizing devices to suit someone's unique needs.
+
+## Inclusively Designing Hackathons and Hacking
+
+It's been said about hackathons that the connections and ideas that get made between people attending are more important than any products that get produced. If hackathons are about building community, issues of inclusion, exclusion and audience diversity are paramount to their design.
+
+In thinking about inclusive design for hackathons, we must ask ourselves some of the same questions we ask in other contexts:
+
+### Asking Who Is Excluded
+
+We must consider <a href="WhatIsTheApproach.md#whoIsExcluded">who might be excluded?</a>, and how to redesign our approach to running, promoting and recruiting for our events.
+
+This is especially critical for hackathons trying to tackle social challenges faced by identifiable groups such as people with disabilities, under the principle of <a href="https://en.wikipedia.org/wiki/Nothing_About_Us_Without_Us" target="_blank" class="link-external">*Nothing About Us Without Us*</a>. Any extra work involved in broadening the reach of the event beyond the "natural" demographic will be balanced out by better outcomes and learning.
+
+### Expanding Collaboration
+
+Physical colocation is common with hackathons, but a requirement to be physically present to participate can exclude people based on ability to travel to the location or by disability.
+
+When considering the design of a hackathon-style event, consider:
+
+* Can you support remote collaboration using audio and video conferencing tools, shared document workspaces, and/or collaborative coding environments?
```

Comment:
Can you suggest any of these, especially ones that are open source or freely web-based (maybe things like talky.io, etc.)